### PR TITLE
Remove dependency on third party http

### PR DESCRIPTION
**Pull request by stanfieldr**

### CODE REVIEW

**Review by jessekoska — APPROVED** [review]

**Comment by marshall-davis** [comment]
Does this not actually *add* a dependency (`axios`) and remove configurability? It seems like you could already pass an axios instance to the constructor and override the default Vue.http.

**Comment by stanfieldr** [comment]
Haha good point, but we're prepping for Vue 3 and Vue Resource is incompatible. Plus this helps know exactly the format of the response sent in as the argument to the promise as well

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nova-tables",
-  "version": "1.8.3",
+  "version": "1.8.4",
```
Comment (marshall-davis) [inline package.json]: If nothing else this is the wrong version increment. This change is not backwards compatible as the http instance can no longer be configured. This must at least be version 1.9.0.

**Review by marshall-davis — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (stanfieldr) [reply package.json]: updated

**Pull request merged**